Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'http://rubygems.org'
```

Comment:
I read the documentation of Gemfile. Should we use https?

Reply:
Hey @ktp242 

Yes, you're 100% correct about using https, not http.
I'll correct it, thank you.

We don't need a `do` on that line - let's talk about it!